lib/payloadAction.ts

```diff
@@ -136,6 +136,17 @@ export const handlePayloadActions = async (
 					await setIsQueueWindowActive(read, persistence, rid, false);
 					await setQueuedMessage(read, persistence, rid, '');
 					console.debug({ rid }, 'Queue Window dropped');
+				} else if (actionName === ActionIds.FEEDBACK_SURVEY) {
```

Comment:
@bhardwajaditya can we add a log point here when we are setting the customField for the feedbackSurvey?  Thanks.